tapir/coop/models.py

```diff
@@ -26,6 +26,10 @@ class ShareOwner(models.Model):
     represent a person or company that does not have their own account.
     """
 
+    MEMBER_STATUS_SOLD = "Sold"
```

Comment:
Maybe make these lower-case constants and put them in a separate (inner?) class (like `ShiftUserCapability`). Have a separate constant that maps them to their ugettext_lazy translations (see `ShiftUserCapability` again)